tensorboard/uploader/uploader_main.py

```diff
@@ -530,12 +530,16 @@ def execute(self, server_info, channel):
                 ("Id", experiment.experiment_id),
                 ("Created", util.format_time(experiment.create_time)),
                 ("Updated", util.format_time(experiment.update_time)),
-                ("Scalars", str(experiment.num_scalars)),
                 ("Runs", str(experiment.num_runs)),
                 ("Tags", str(experiment.num_tags)),
+                ("Scalars", str(experiment.num_scalars)),
             ]
+            if experiment.total_blob_bytes:
+                data.append(
+                    ("Binary object bytes", str(experiment.total_blob_bytes))
```

Comment:
Let’s avoid whitespace in the field name. As described in #2941, this
format is designed to be easily parseable with something like:

```
tensorboard dev list |
    awk '$1 == "Id" { id = $2 } $1 == "Scalars" && $2 > 1000 { print id }'
```

With spaces in `"Binary object bytes"`, the `$1 == "Scalars"` would have
to be something like `($1 " " $2 " " $3) == "Binary object bytes"`,
which is pretty ugly.

Reply:
I think it's more important that this format be clearly intelligible to humans than that it be machine parseable.  If we want to make it easier to parse, we should add some non-space delimiter between the field name and the value (like `:`) or (my preference), just add an actually standard output format option like JSON, rather than artificially constraining how much information we provide to users.

There was already an internal thread about the naming to try to determine what to call this field (to avoid "blob") and I don't think there is a one-word name that is sufficiently unambiguous since we need to specify that the unit here is bytes rather than the number of blobs or blob sequences.

Reply:
Either of those options—an unambiguous delimiter (`:` is kind of awkward
due to the URLs) or a `--json`/`--output json` flag—works for me, too.
Currently (in master), the output is clear to both humans and machines,
so I would prefer not to regress from that.

Reply:
Thanks for the discussion. I think "Binary object bytes" Is a nice tradeoff balance clarity and conciseness.

I like the `--json` option better than `:`. `:` feels a little awkward as @wchargin  mentioned. Do you want me to add the json mode in this PR? It feels like a sufficiently distinct feature that merits a separate PR.

Reply:
Thanks, @GalOshri . I'll merge the PR as is soon.

I filed b/153232102 to track the proposed JSON output feature. It is sufficiently distinct from the goal of this PR to merit a separate PR.

Reply:
@wchargin I'm not sure if you want to take a look again at this PR. If you do, please let me know.

Reply:
Yes, I’ll take a look at the `--json` change and then revisit here.
Thanks!